Source/Streamstone/Stream.Operations.cs

```diff
@@ -31,7 +31,7 @@ public Stream Execute()
 
                 try
                 {
-                    table.Execute(insert.Prepare());
+                    table.ExecuteAsync(insert.Prepare()).Wait();
```

Comment:
I just tried to not change the current public APIs, but I still think the sync code should be removed...